ui/src/actions/apps.js

```diff
@@ -36,3 +36,27 @@ export const deleteApp = name => fetchAction(
   [APP_DELETE_REQUEST, APP_DELETE_SUCCESS, APP_DELETE_FAILURE],
   async () => DataService.deleteApp(name),
 )();
+
+export const registerPlatform = (platform) => ({
+  type: 'platform/REGISTER',
```

Comment:
a very small thing: can we follow the same convention of types in this file, use "_" instead of "/"?